test/test_profiler.py

```diff
@@ -303,6 +303,36 @@ def test_export_stacks(self):
                     pass
                 assert is_int, "Invalid stacks record"
 
+    @unittest.skipIf(not kineto_available(), "Kineto is required")
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA is required")
+    def test_tensorboard_trace_handler(self):
+        with profile(use_cuda=True, use_kineto=True):
+            self.payload()
+
+        import os,time,tempfile,json
```

Comment:
move to top of the file, merge with existing imports in alphabet order?
also, better to use multiple lines style to align with existing imports.